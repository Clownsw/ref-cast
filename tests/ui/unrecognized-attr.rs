use ref_cast::RefCast;

#[derive(RefCast)]
#[repr(transparent)]
#[unrecognized]
struct Test {
    s: String,
}

fn main() {}
