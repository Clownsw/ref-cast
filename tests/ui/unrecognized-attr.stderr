error: unrecognized attribute on struct that implements RefCast
 --> $DIR/unrecognized-attr.rs:5:1
  |
5 | #[unrecognized]
  | ^^^^^^^^^^^^^^^

error: cannot find attribute `unrecognized` in this scope
 --> $DIR/unrecognized-attr.rs:5:3
  |
5 | #[unrecognized]
  |   ^^^^^^^^^^^^
